app/static/js/admin/guest/explore/event-box-component.js

```diff
@@ -22,7 +22,12 @@ Vue.component('event-box', {
             return moment(this.event.start_time).format('ddd, MMM DD HH:mm A');
         },
         locationName: function () {
-            return this.event.location_name.split(",")[0];
+            if(this.event.location_name){
+                return this.event.location_name.split(",")[0];
+            }
+            else{
```

Comment:
Please follow the indentation rule of jslint to put `else` on the same line as closing bracket of `if`. Also provide spaces before opening brackets.